src/web/utils/formValidation.js

```diff
@@ -1,4 +1,35 @@
+import _ from 'lodash';
+
 const formValidation = {
+  checkInputValidity: (input, customErrorMessage) => {
```

Comment:
It looks like the value for text fields is cleared when when the validation is triggered. See example gif below. Is this intentional? I think we should consider keeping the value present, so the user can inspect what they entered to confirm the issue (eg. check if they only partially copied a key or secret).

![Jul-10-2019 17-01-13](https://user-images.githubusercontent.com/3423655/61008772-253fa400-a336-11e9-9533-081535bdf1bc.gif)

Reply:
_Carry over from PMs for posterity._

This rendered a centered form makes me think there is a caching issue locally for you, because these styles were removed. I am also unable to replicate the disappearing fields in the latest Firefox and Chrome.